### PR TITLE
Enhanced the styling of the footer 

### PR DESCRIPTION
**Pull request by VyjayantiN**
Summary
This pull request enhances the styling of the footer section, improving its visual appearance and user experience.

Changes Made
Rearranged the content sections within the footer for better organization and readability.
Applied updated HTML structure and semantic tags to improve accessibility and maintainability.
Modified the layout of the footer, including the order of content sections.
Implemented revised CSS styles to enhance the overall styling of the footer.
Adjusted spacing, alignment, and font styles to create a more visually appealing footer.
Screenshots

image

### CODE REVIEW

**Comment by VyjayantiN** [comment]
@codervivek5 Please look into it!!

**Comment by VyjayantiN** [comment]
@codervivek5 Please look into it!!

**Comment by VyjayantiN** [comment]
@codervivek5 Could you please look into this!!

File: index.html [hunk 1]
```diff
@@ -10,18 +10,19 @@
     <link rel="stylesheet" href="style/review.css">
     <!-- bootstrap css -->
     <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/5.3.2/css/bootstrap.min.css"> -->
-
+    
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
     <title>VigyBag | Home</title>
-    <link rel="icon" type="image/x-icon" href="/img/logo.png">
```
Comment (codervivek5) [inline index.html:17]: why you remove this ???

**Comment by codervivek5** [comment]
@VyjayantiN I think it looks fine but make some changes.

- shift quick links in the left side
- shift the left section to the middle including social links then it will look fine.

**Comment by codervivek5** [comment]
before make any changes pull the code first

**Comment by codervivek5** [comment]
reduced the height of the footer section also...if possible

**Comment by VyjayantiN** [comment]
@codervivek5 Please look into this.
image

**Comment by VyjayantiN** [comment]
@codervivek5 I made the changes which you told. Please look into .

File: index.html (hunk 1 above)
Comment (VyjayantiN) [reply index.html:17]: The dark color text in the logo will not be visible on the dark footer. So I made a light colored logo.

**Comment by codervivek5** [comment]
> @codervivek5 Please look into this. image


shift copywrite section below to the social media icons....and the others is perfect

**Comment by codervivek5** [comment]
Hello ! @VyjayantiN any update ???

**Comment by VyjayantiN** [comment]
@codervivek5   I am extremely sorry!! 
I am having semester exams going on right now.
I can contribute after they end. Sorry for the inconvenience.

**Comment by codervivek5** [comment]
no worry......when will you be free kindly let me know about that

**Pull request closed without merging**